src/vs/workbench/parts/preferences/electron-browser/settingsEditor2.ts

```diff
@@ -844,7 +858,11 @@ export class SettingsEditor2 extends BaseEditor {
 		if (this.settingsTreeModel) {
 			this.settingsTreeModel.update(resolvedSettingsRoot);
 
-			return this.renderTree(undefined, forceRefresh);
```

Comment:
Calling this before settingsTreeModel has been created doesn't seem to cause any issue, but I put this here just in case, replacing renderTree.